c/array-tiling/mbpr2_true-unreach-call.c

```diff
@@ -0,0 +1,49 @@
+extern void __VERIFIER_error() __attribute__ ((__noreturn__));
+extern void __VERIFIER_assume(int);
+void __VERIFIER_assert(int cond) { if(!(cond)) { ERROR: __VERIFIER_error(); } }
+
+#define CELLCOUNT
+
+void main()
```

Comment:
Perhaps this is a nitpick, but the prototype of main should be `int main(void)` in this case. From the C standard:

```
It shall be defined with a return type of int and with no parameters:

int main(void) { /* ... */ }

or with two parameters (referred to here as argc and argv, though any names may be
used, as they are local to the function in which they are declared):

int main(int argc, char *argv[]) { /* ... */ }

or equivalent;
```

Reply:
Ok. Thanks. I will upload the updated files.